src/main/module/module-info.java

```diff
@@ -2,6 +2,7 @@
 	requires org.junit.jupiter.api;
 	requires org.junit.jupiter.params;
 	requires org.junit.platform.commons;
+	requires org.junit.platform.launcher;
 
```

Comment:
This needs to be here for services to work on the module path:

```
	provides org.junit.platform.launcher.TestExecutionListener
			with org.junitpioneer.jupiter.issue.IssueExtensionListener;
```

(I wanted to propose a change, but I can't propose one at the end of the file, which is where this should go.)